src/streamdatasetjson/dataset.py

```diff
@@ -1,34 +1,45 @@
+from typing import Iterable, List, NewType
 import utils
 
+
+Row = NewType("Row", List[str])
+VarMeta = NewType("VarMeta", dict)
+
 class Dataset:
-    def __init__(self, dataset_json_file, name):
+    """
+    Dataset. Should not be instantiated manually.
+    """
+    def __init__(self, dataset_json_file, name, prefix):
```

Comment:
Can you please add types
(self, dataset_json_file: str, name: str, prefix: str)